### DISCUSSION TITLE
Controlling one specific drone in a swarm using an RC transmitter

### DISCUSSION THREAD

**Discussion by ZakariaLaakel** [ctu-mrs/mrs_uav_system#96] [Q&A] [answered]
Hello,

I have currently a swarm of F450 drones that I arm and trigger the offboard via an RC transmitter (Hitec flash 8). This all happens simultaneously (different RC receivers, Optima SL, all bound to 1 RC transmitter). However, I was wondering how it was possible to control 1 specific UAV of the swarm, for example in case of emergency. At the moment every switch I trigger has an impact on all the different UAVs, and this is not always what I want. Is there a way to manually control 1 UAV using the transmitter without impacting the other UAVs of the swarm?

Thank you for the help!

Zakaria

**Comment by DanHert** [comment, marked as answer]
Hello, we do not currently have a way to control one specific UAV with the RC.  However, you can still control a specific UAV through ROS, either by calling services/publishing topics or by using the mrs_uav_status - https://github.com/ctu-mrs/mrs_uav_status

Dan H.
